### DISCUSSION TITLE
Web Builder v0.9g

### DISCUSSION THREAD

**Discussion by LEOOZERAN** [grblHAL/core#611] [Q&A]
In the Web Builder v0.9g firmware for the Devtronic_CNC_Controller_V2_ board (STM32F411), the firmware is not generated. When turning on Spindle sync and Log for configuration. Without them, the firmware is generated. FIX THIS PROBLEM! PLEASE!

**Comment by phil-barrett** [comment]
Better to post on [this thread](https://github.com/grblHAL/core/discussions/203).  It's long but centralizes all the web builder issues.
